src/lib/plugins/column-filter/column-filter.component.ts

```diff
@@ -140,4 +157,70 @@ export class ColumnFilterComponent implements OnInit {
 		this.context.$implicit.items = [];
 		this.vscrollContext.container.reset();
 	}
+
+	clear() {
+		this.search = '';
+		this.exprValue = [];
+		this.context.$implicit.reset.execute();
+	}
+
+	get operatorTemplateKey() {
+		const { column, operator } = this;
+		switch (operator) {
+			case 'contains': {
+				return 'plugin-column-filter-default-contains.tpl.html';
+			}
+			case 'between': {
+				return column.type === 'date'
+					? 'plugin-column-filter-date-between.tpl.html'
+					: 'plugin-column-filter-default-between.tpl.html';
+			}
+			case 'isEmpty':
+			case 'isNotEmpty':
+			case 'isNull':
+			case 'isNotNull': {
+				return 'plugin-column-filter-default-disabled.tpl.html';
+			}
+			default: {
+				return column.type === 'date'
+					? 'plugin-column-filter-date.tpl.html'
+					: 'plugin-column-filter-default.tpl.html';
+			}
+		}
+	}
+
+	get operatorValue() {
+		switch (this.operator) {
+			case 'isNull':
+			case 'isNotNull':
+			case 'isEmpty':
+			case 'isNotEmpty': {
+				return null;
+			}
+			case 'between': {
+				return this.column.type === 'date'
+					? this.exprValue.map(d => this.yyyymmdd(d, '-'))
+					: this.exprValue;
+			}
+			default: {
+				return this.column.type === 'date'
+					? this.yyyymmdd(this.exprValue[0], '-')
+					: this.exprValue[0];
+			}
+		}
+	}
+
+	get hasOperators() {
+		return this.operators && this.operators.length > 1;
+	}
+
+	yyyymmdd(date, separator) {
```

Comment:
I think we have something for this, pliease look at date editors that we have in cells